go.mod

```diff
@@ -46,6 +46,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.11.0-beta.0.0.20211208212546-f236f0345ad2
 )
 
+require k8s.io/pod-security-admission v0.0.0
```

Comment:
@Madhu-1 do you understand why this is moved from `// indirect` to a direct requirement? I guess it should become part of the section above it...

Reply:
https://github.com/ceph/ceph-csi/pull/3485#pullrequestreview-1163027598 addresses it, this can be recreated with dependabot after that one is merged

Reply:
it's automatically done as part of the `go mod tidy` command its used in the E2E not sure how it was missed in earlier PRs, yes we can move it to existing required section.